www/i18n/en.json

```diff
@@ -487,9 +343,7 @@
         "no-nrelop-start": "OPcode {{token}} does not start with 'nrelop'",
         "not-enough-parts": "OPcode {{token}} does not have at least three '_' characters",
         "invalid-subgroup": "Invalid OPcode {{token}}, subgroup {{subgroup}} not found in list {{config_subgroups}}",
-        "invalid-subgroup-no-default": "Invalid OPcode {{token}}, no subgroups, expected 'default' subgroup",
```

Comment:
we do still use this for studies that support subgroups, such as Denver CASR

Reply:
There are two places where `invalid-subgroup` is used, looking back through the GitBlame [here](https://github.com/e-mission/e-mission-phone/blame/1ebf648f5a8d14d857b9a191b6a7ccca5a3c6291/www/js/config/dynamic_config.js) it looks like the 2nd one used to be the "no-default" text, should I switch `invalid-subgroup` for `invalid-subgroup-no-default` in [this case](https://github.com/e-mission/e-mission-phone/blob/aa996b24ea22ba7ba353d98286b86e9b4021e793/www/js/config/dynamic_config.js#L241) and restore the key?